.github/workflows/check-vulnerability-whitesource.yml

```diff
@@ -21,29 +40,62 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: ${{ matrix.java }}
-
       - name: Vulnerability Scan
+        id: vulnerability_scan
         env:
           wss_apikey: ${{ secrets.WSS_API_KEY }}
         run: |
-          cd standalone-tools
-          mvn -v
-          npm -v
-          gradle -v
-          yarn -v
+          export PATH=$JAVA_HOME:$PATH
+          cd standalone-tools ; ls -ltr 
+          sudo yum install -y maven
+          wget -q https://services.gradle.org/distributions/gradle-6.7-bin.zip
+          sudo mkdir /opt/gradle
+          sudo unzip -d /opt/gradle gradle-6.7-bin.zip
+          wget -qO - https://rpm.nodesource.com/setup_14.x | sudo bash -
+          sudo yum install -y nodejs
+          wget -qO - https://dl.yarnpkg.com/rpm/yarn.repo | sudo tee /etc/yum.repos.d/yarn.repo
+          sudo rpm --import https://dl.yarnpkg.com/rpm/pubkey.gpg
+          sudo yum install yarn -y
+          export PATH=$PATH:/opt/gradle/gradle-6.7/bin
+          gradle -v; mvn -v ; npm -v; yarn -v
```

Comment:
Would it better for us to create an AMI that has all the dependencies already installed?